scripts/raceday_verify.py

```
@@ -0,0 +1,848 @@
+
+def _find_latest_report(race_date):
+    """Load the most recent raceday report JSON for comparison."""
+    report_dir = Path("data")
+    if not report_dir.exists():
+        return None
+    patterns = sorted(report_dir.glob(f"raceday_report_{race_date}_*.json"), reverse=True)
+    if not patterns:
+        return None
+    try:
+        return json.loads(patterns[0].read_text(encoding="utf-8"))
+    except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)